BitmovinConvivaAnalytics/Classes/Helper/AdEventUtil.swift

```diff
@@ -13,6 +13,18 @@ import Foundation
 enum AdEventUtil {
     static let positionRegexPattern = "pre|post|[0-9]+%|([0-9]+:)?([0-9]+:)?[0-9]+(\\.[0-9]+)?"
 
+    static func parseAdPosition(event: AdBreakStartedEvent, contentDuration: TimeInterval) -> ConvivaSDK.AdPosition {
```

Comment:
Why do we need both now? 🤔 

I don't understand why we need the information multiple times. If there is a pre-roll AdBreak, every Ad in there will be reported as pre-roll as well right?

Reply:
The documentation is not very clear on this part. The Android integration only track metadata in the `AdStarted`.

Revisiting this, I will drop this again.

Reply:
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/68/commits/c85be19d7758a462fcbcb383a04c49dc75e39884
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/68/commits/49b1f8a998af3c6c74b357e53d469bbbd2f73dd6